source/macros.tex

```diff
@@ -156,6 +156,7 @@
 
 % Code and definitions embedded in text.
 \newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\mcode}[1]{\text{\tcode{#1}}}   % \tcode in math mode
```

Comment:
@jensmaurer: I agree. If `\tcode` already works out of the box, we should just use that, never mind the name. So for example we can say `$\tcode{T}_i$` rather that `\tcode{T}$_i$`.